roaringbitmap/src/main/java/org/roaringbitmap/buffer/MappeableBitmapContainer.java

```diff
@@ -1269,27 +1269,26 @@ public int nextSetBit(final int i) {
   }
 
   /**
-   * Find the index of the next unset bit greater or equal to i, returns -1 if none found.
+   * Find the index of the next clear bit greater or equal to i.
    *
    * @param i starting index
-   * @return index of the next unset bit
+   * @return index of the next clear bit
    */
-  public short nextUnsetBit(final int i) {
-    int x = i / 64;
+  public int nextClearBit(final int i) {
```

Comment:
I'm not sure if anyone is using this. I prefer the new name though.

Reply:
I slightly changed the semantics (return `highestBit() + 1` instead of `-1`), its in line with `BitSet` now.

It was unused inside the library. Could keep both maybe?

Reply:
No I think the new name is a good idea.